### ISSUE TITLE
Traditional Chinese homepage has a prominent broken link, and all translated homepages may not have been updated

### ISSUE THREAD

**Issue by thepegisin** [cagov/covid19#1145]
On the homepage of our Traditional Chinese site, the link to the **County variance** page is missing a slash, which makes it a broken link. Please fix.

<img width="898" alt="2020-07-30_18-02-42" src="https://user-images.githubusercontent.com/63249920/88988938-2ddf8100-d28f-11ea-9e18-9a2c471e8158.png">

Also all translated homepages don't appear to have the current version of the square button links on the bottom of the page. They should be:

County information (variance) - Testing (and treatment) - Industry guidance - (Stay home) Q&A

<img width="845" alt="2020-07-30_18-08-07" src="https://user-images.githubusercontent.com/63249920/88989186-ec9ba100-d28f-11ea-8b77-b603e70aa4b8.png">

Instead, it's (on Spanish homepage):

Roadmap - County information (variance) - Industry guidance - (Stay home) Q&A

<img width="850" alt="2020-07-30_18-11-11" src="https://user-images.githubusercontent.com/63249920/88989348-40a68580-d290-11ea-9009-905daaa31657.png">

It's likely that's true on all the translated pages, as the County page also appears second in the Chinese Traditional homepage.

In summary:
- [x]  Fix County link on Chinese Traditional page
- [x]  Update button links/order on all translated pages
- [x]  Check for other elements on the English homepage that didn't get updated on translated pages

**Comment by thepegisin** [comment]
As of today (8/13/2020), this is the last broken link on the site! Let's go!

**Comment by thepegisin** [comment]
Fixed!
